demo/app.js

```diff
@@ -502,10 +523,38 @@ const Demo = React.createClass({
     });
   },
 
+  _nextSibling () {
+    console.log('next');
+  },
+
+  _previousSibling () {
+    console.log('previous');
+  },
+
+  _toggleDrawer () {
+    this.setState({
+      showDrawer: !this.state.showDrawer
+    });
+  },
+
   render () {
+    const navContent = {
+      duration: 200,
+      onNextClick: this._nextSibling,
+      label: _find(this.state.drawerSiblings, { selected: true }).id + ' of ' + this.state.drawerSiblings.length,
+      onPreviousClick: this._previousSibling
```

Comment:
following our naming conventions it should be prefixed with `_handle`. `_handlePreviousSiblingClick`